webrtc/README.md

```diff
@@ -0,0 +1,30 @@
+# webrtc vosk-server  
+
+
+## Setup environment and run it. 
+### Set model path  
+Setup path to ./model  
+The models can be download from here https://alphacephei.com/vosk/models   
+
+### python environment   
+The sample can work in python 3.8 
+$ pip install numpy aiortc aiohttp aiorpc vosk  
```

Comment:
I would prefer to not depend on numpy

Reply:
<img width="1405" alt="Screen Shot 2021-02-21 at 4 37 22 AM" src="https://user-images.githubusercontent.com/12568287/108608001-d48f9880-73fe-11eb-9dc2-f86fe7b095cb.png">
<img width="652" alt="Screen Shot 2021-02-21 at 4 37 31 AM" src="https://user-images.githubusercontent.com/12568287/108608002-d6595c00-73fe-11eb-9cc5-0414ee11bb5f.png">
I install numpy module is because of the frame.to_ndarray() need to use numpy module.

Reply:
@nshmyrev 

```python
frame = await self.__track.recv()
```
MediaStreamStrack  .recv() return AudioFrame here
Ref: https://aiortc.readthedocs.io/en/latest/api.html#aiortc.MediaStreamTrack

```python
frame = self.__resampler.resample(frame)
```
Input and output are both AudioFrame 
Ref: https://pyav.org/docs/stable/api/audio.html#av.audio.resampler.AudioResampler

```python
data = frame.to_ndarray()
```
But right now, AudioFrame's method only can get numpy array first. 
Ref: https://pyav.org/docs/stable/api/audio.html#module-av.audio.frame 

<img width="816" alt="Screen Shot 2021-02-21 at 11 41 29 AM" src="https://user-images.githubusercontent.com/12568287/108614928-e4789e00-7439-11eb-898a-56c1e9ad7475.png">

In document, it show that Numpy must be installed. 

I am attempting to find out another way to get bytes from AudioFrame without using Numpy object.

Reply:
Hi @nshmyrev  
Thanks for your suggestion. 

The code is not depend on numpy right now.  https://github.com/alphacep/vosk-server/pull/94/commits/167396014d505449f284c53582437d062788fbab

And I also update README https://github.com/alphacep/vosk-server/pull/94/commits/5ddf34258cf397a05a9afd2b4c80bbb989a80848